docs/generate.edn

```diff
@@ -30,6 +30,33 @@
     .f (ToString))
    true))
 
+(defn print-type-info [buffer]
+  (-> (ExpectTable)
+      (| (Take "name") (ExpectString) >= .p_type_name)
+      (| (Take "type") (ExpectInt) >= .p_type_type)
+      "[`" >> buffer .p_type_name >> buffer "`](" >> buffer
+      .p_type_type
+      (Match
+       [2; Enum
```

Comment:
Don't we have an enum for this? (Type.Enum or something?)
Maybe we can add one?

Reply:
We do have a [`Type`](https://docs.fragcolor.xyz/reference/enums/Type/) enum. I'll try to see if I can use it.